bin/cli.js

```diff
@@ -193,17 +196,16 @@ const getConfig = () => {
 
   config.files = merge(config.files, program.args);
   config.files = loadFiles(config.files);
-
   config.exclude = merge(config.exclude, program.exclude);
   config.files = exclude(config.files, config.exclude);
   config.nodeOnly = merge(config.nodeOnly, program.nodeOnly);
   config.browserOnly = merge(config.browserOnly, program.browserOnly);
+  config.verbosity = program.verbose;
 
   if (config.environments.includes('browser')) {
     config.browser = config.browser || {};
-
-    config.browser.browsers = merge(
-      config.browser.browsers, program.browsers);
+    config.browser.browsers = merge(config.browser.browsers, program.browsers);
+    if (program.verbose) config.browser.logLevel = logLevels[program.verbose];
```

Comment:
I'd propose to do the opposite and have `browser-log` overwrite `verbose` for browser. This way it will be possible to have separate log level for browser if needed (by setting both `verbose` and `browser-log`), wdyt?

Reply:
And it does it on the next line, though it will be better to merge those lines.